LibraryApp/src/app/components/LoadingHOC/index.js

```diff
@@ -5,7 +5,11 @@ import { ActivityIndicator } from 'react-native';
 
 import styles from './styles';
 
-const LoadingHOC = Component => ({ loading, ...props }) =>
-  loading ? <ActivityIndicator style={styles.container} /> : <Component {...props} />;
+const LoadingHOC = Component => ({ loading, notFlex, ...props }) =>
+  loading ? (
+    <ActivityIndicator style={[styles.container, notFlex && styles.notFlex]} />
```

Comment:
I would use `flex` instead of `notFlex` and make default style not be flex 1

Reply:
it makes a lot of sense 😅